src/test/scala/uk/gov/hmrc/perftests/disareturns/AuthRequests.scala

```diff
@@ -16,13 +16,15 @@
 
 package uk.gov.hmrc.perftests.disareturns
 
-import io.gatling.core.Predef._
-import io.gatling.http.Predef._
-import io.gatling.http.request.builder.HttpRequestBuilder
-import uk.gov.hmrc.performance.conf.ServicesConfiguration
+import play.api.libs.ws.DefaultBodyWritables.writeableOf_String
+import play.api.libs.ws.ahc.StandaloneAhcWSClient
 import uk.gov.hmrc.perftests.disareturns.constant.AppConfig.ggSignInUrl
 
-object AuthRequests extends ServicesConfiguration {
+import scala.concurrent.Await
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Relying on global is fine for small scripts, but for injected classes it’s better to inject ExecutionContext explicitly

`(ws: StandaloneAhcWSClient)(implicit ec: ExecutionContext)`